packages/fcl/src/wallet-utils/on-fcl-message.js

```diff
@@ -0,0 +1,4 @@
+export const onFclMessage = (callback = () => {}) => {
```

Comment:
I was thinking it could have an interface like this:
```javascript
onFclMessage("FCL:VIEW:READY:RESPONSE", message => { console.log(message) })
```
thoughts?

Reply:
For sure. Had that then pulled to not leave dangling log 👍🏼

Reply:
Updated and added a check for expected reply & data